New layout and sections (#51)
dandi/handbook: docs/13_upload.md

# Uploading Data

## Creating an Account on DANDI
In order to create a new `Dandiset` and upload your data, you need to have a DANDI account.

> To create a DANDI account you need:

Reviewer: ```suggestion
To create a DANDI account:
```

1. [Create a GitHub account](https://github.com/) if you don't have one.
2. Using your GitHub account, [register a DANDI account](https://gui.dandiarchive.org/#/user/register).

You will receive an email acknowledging activation of your account within 24
hours, after which you can log in to DANDI using GitHub by clicking the login
button.

## Creating Dandiset and adding the data

> DANDI provides a production server (http://dandiarchive.org/) for real data and an identical staging server 
> (https://gui-staging.dandiarchive.org/) for test data.

Reviewer: ```suggestion
You can create a new Dandiset at http://dandiarchive.org . This Dandiset can be fully public or embargoed according to NIH policy.
```

When you create a Dandiset, a permanent ID is automatically assigned to it.

> To prevent the production server from being inundated with test Dandisets, we encourage developers to develop 
> against the staging server. The below instructions will alert you to where the commands for interacting with these 

Reviewer: ```suggestion
To prevent the production server from being inundated with test Dandisets, we encourage developers to develop 
against the development server (https://gui-staging.dandiarchive.org/). Please note that the development server
should not be used to stage your data. All data are uploaded as draft and can be adjusted before publishing on
the production server. The development server is primarily used by users learning to use DANDI or by developers.

The below instructions will alert you to where the commands for interacting with these 
```

    If the report is too large to efficiently navigate in your console, a report can be saved using

            nwbinspector <source_folder> --config dandi --report-file-path <report_location>.txt
        
    4. Once your files are confirmed to adhere to the Best Practices, perform an official validation of the NWB files by running: `dandi validate <source_folder>`. 
        **If you are having trouble with validation, make sure the conversions were run with the most recent version of `PyNWB` and `MatNWB`.**
    5. Prepare a dataset folder for upload:

            dandi download https://dandiarchive.org/dandiset/<dataset_id>/draft
            cd <dataset_id>
            dandi organize <source_folder> -f dry
            dandi organize <source_folder>
            dandi upload

>             For upload to staging server, specify that explicitly via `-i` option, e.g.

Reviewer: ```suggestion
            For upload to the development server, specify that explicitly via `-i` option, e.g.
```

            `dandi upload -i dandi-staging`.

    6. Add metadata by visiting your Dandiset landing page: 
       `https://dandiarchive.org/dandiset/<dataset_id>/draft` and clicking on the `METADATA` link.
    7. Use the Dandiset URL in your preprint directly, or download it using the DANDI CLI:
            `dandi download https://dandiarchive.org/dandiset/<dataset_id>/draft`.

>         [Reach out to Dandi team for help](https://github.com/dandi/helpdesk/discussions) with the BIDS format.

Reviewer: This last sentence seems out of place, and also because bids support i think is almost there, hence should be updated to reflect bids datasets.

Author: will check, I mostly copied this text
